app/controllers/api/v1/assessments_controller.rb

```diff
@@ -11,25 +11,26 @@ class Api::V1::AssessmentsController < ApiController
         key :api_token, []
       end
       key :tags, ['Assessments']
-
-      parameter do
-        key :name, :q
-        key :in, :path
-        key :description, 'query to execute'
-        key :required, true
-        key :type, :string
-      end
+      parameter name: :query, in: :path, type: :string, required: true,
+                description: 'the query string to search for'
+      parameter name: :page, in: :path, type: :int, required: false,
+                description: 'the page index to return, the first page is 1'
+      parameter name: :per_page, type: :int, required: false,
+                description: 'how many assessments should be returned for a page'
```

Comment:
Does this need to be `in: :path` as well?

Reply:
yeah, but it should actually be `in: :query` - I'll add to that and update the others